esaco-app/src/main/java/it/infn/mw/esaco/web/TokenControllerUtils.java

```diff
@@ -0,0 +1,31 @@
+package it.infn.mw.esaco.web;
+
+import java.text.ParseException;
+
+import org.springframework.web.bind.annotation.RestController;
+
+import com.google.common.base.Strings;
+import com.nimbusds.jwt.JWTParser;
+
+import it.infn.mw.esaco.exception.TokenValidationException;
+
+@RestController
+public class TokenControllerUtils {
```

Comment:
TokenControllerUtils could have also been defined as an interface with [default methods](https://docs.oracle.com/javase/tutorial/java/IandI/defaultmethods.html).
This comment is not to suggest a change, but just to share that a different approach is also possible